dev-guide/cluster-version-operator/dev/upgrades.md

```diff
@@ -58,6 +58,8 @@ Which in practice can be described in runlevels:
 0000_90_*: reserved for any post-machine updates
 ```
 
+**Note**: During the release process if the component's run level is not specified i.e. the prefix is `0000_00_`, it will be assigned to `0000_50_*` during the OCP release process. This is done to make sure if an component is agnostic about the run level it runs at, it is run as part of `0000_50_*` as it helps us to update more components in parallel which helps in reducing the overall upgrade time.
```

Comment:
> is not specified i.e. the prefix is `0000_00_`

Is *this* really the case? I'd expect this to be a `00` runlevel. Maybe we should suggest an entirely different, simpler prefix such as `00_` that @wking used in https://github.com/openshift/api/pull/1541?

Reply:
You are right, anything other than `0000_` would trigger this code, which is responsible for the renames or re-assignment https://github.com/openshift/oc/blob/13225e00caf1ad2d3603e1d1cc8651833f2effcb/pkg/cli/admin/release/new.go#L1412-L1416

Reply:
From [the implementation][1], opening with `0000_` is saying "I want to explicitly declare a runlevel".  The advice for operator maintainers is:

* Many of you don't care about your runlevel, so don't prefix with `0000_`.
* A few of you do care about runlevel, and you should match the `0000_<runlevel>_<dash-separated-component>_<manifest_filename>` template described in `operators.md` (a sibling to the file you're touching here).
* Whether or not you care about runlevel, the remainder of your filename will be sorted, so pick a name like `12_whatever.yaml` if you need to sort after earlier manifests or before later manifests within the rest of your component's manifests.

[1]: https://github.com/openshift/oc/blob/13225e00caf1ad2d3603e1d1cc8651833f2effcb/pkg/cli/admin/release/new.go#L1412-L1416

Reply:
Updated the text in https://github.com/openshift/enhancements/commit/6b214735375cb686dddced796961a49d51e1494d as per the discussion